src/styles/globals.css

```
@@ -671,3 +728,5 @@
   fill: theme(colors.zinc.600);
 }
 
+
+@import "../app/map/Canvas/OperationOverlay/_styles/overlay.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)